# Structure for packets sent from the user interface to the PathPlanner
# Sends the uuid of the program to be executed

uint32 program_run_id
uint32 run_waypoint_id
string action           # type of action to be sent this is kind of unnecessary, maybe switch with datapoint id?

---

string status # Whether or not the data collection was successful
# string data # A json string containing:
#                    {
#                        type: <img>, <environ>,
#                        data: {
#                            [filepath: <absolute_path>]
#                            [temp: float,
#                            humidity: float,
#                            CO2: float]
#                        }
#                    }